src/sentry/static/sentry/app/components/repositoryProjectPathConfigForm.tsx

```diff
@@ -91,6 +92,20 @@ export default class RepositoryProjectPathConfigForm extends React.Component<Pro
     ];
   }
 
+  handlePreSubmit() {
+    trackIntegrationEvent(
+      {
+        eventKey: 'integrations.stacktrace_submit_config',
+        eventName: 'Integrations: Stacktrace Submit Config',
+        setup_type: 'manual',
+        view: 'integration_configuration_detail',
+        provider: this.props.integration.provider.key,
+      },
+      this.props.organization,
+      {startSession: true}
```

Comment:
@MeredithAnya curious why we are starting the analytics session when you press submit instead of earlier (such as the component loading).

Reply:
I actually think I just remove this, I dont need to be starting a session here